.github/workflows/docs-build.yml

```diff
@@ -41,7 +41,7 @@ jobs:
     name: "Build Toplevel Documentation"
     needs: filter
     if: ${{ needs.filter.outputs.toplevel_changed == 'true' }}
-    uses: open-edge-platform/orch-ci/.github/workflows/build-documentation.yml@b14b0105bf312a2c98457dd021622de4c14f6bff
+    uses: ./.github/actions/docs-build
     secrets:
```

Comment:
`secrets:` should be `with:`, since this is a reusable Action, as opposed to reusable Workflow.